x-pack/platform/plugins/shared/metrics_experience/server/lib/extract_time_series_fields.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FieldCapsFieldCapability } from '@elastic/elasticsearch/lib/api/types';
+
+export function extractTimeSeriesFields(
+  fields: Record<string, Record<string, FieldCapsFieldCapability>>
+) {
+  const numericTypes = [
+    'long',
+    'integer',
+    'short',
+    'byte',
+    'double',
+    'float',
+    'half_float',
+    'scaled_float',
+    'unsigned_long',
+    'histogram',
+  ];
```

Comment:
Perhaps we could use the `ES_FIELD_TYPES` constant from `kbn-field-types` instead